src/payment/data/sagas.js

```diff
@@ -186,7 +186,8 @@ export function* handleFetchClientSecret() {
   try {
     yield put(clientSecretProcessing(true));
     // TODO: possibly add status for stripe elements loading?
-    // const result = yield call(PaymentApiService.getClientSecret);
+    const result = yield call(PaymentApiService.getClientSecret);
+    yield put(clientSecretDataReceived(result));
```

Comment:
Ah I see, I reviewed our PR my commit, disregard my previous comment on updating the store!